test/Microsoft.AspNet.Mvc.IntegrationTests/DictionaryModelBinderIntegrationTest.cs

```diff
@@ -134,6 +137,7 @@ public class DictionaryModelBinderIntegrationTest
         [Theory]
         [InlineData("?[key0]=10")]
         [InlineData("?[0].Key=key0&[0].Value=10")]
+        [InlineData("?index=low&[low].Key=key0&[low].Value=10")]
```

Comment:
> That would ensure validation works correctly in this case. Is that what you're looking for?

Yeah, I want to make sure that we have good tests for each support key-syntax, and that those include some tests where validation errors are triggered.